src/Apps/Artist/routes.tsx

```diff
@@ -56,13 +56,26 @@ graphql`
   }
 `
 
+const ArtistApp = loadable(() => import("./ArtistApp"))
+const OverviewRoute = loadable(() => import("./Routes/Overview"))
+const WorksForSaleRoute = loadable(() => import("./Routes/Works"))
+const AuctionResultsRoute = loadable(() => import("./Routes/AuctionResults"))
+
+// Artist pages tend to load almost instantly, so just preload it up front
+if (typeof window !== "undefined") {
+  ArtistApp.preload()
+}
+
 // FIXME:
 // * `render` functions requires casting
 // * `Redirect` needs to be casted, as it’s not compatible with `RouteConfig`
 export const routes: RouteConfig[] = [
   {
     path: "/artist/:artistID",
-    getComponent: () => loadable(() => import("./ArtistApp")),
+    getComponent: () => ArtistApp,
+    prepare: () => {
+      ArtistApp.preload()
+    },
```

Comment:
If we do decide to always preload, it's likely that we won't need this.

Reply:
Its a no-op if already loaded, so would rather keep things consistent. We might not want to preload immediately in the future.